trio/_core/_generated_run.py

```diff
@@ -78,7 +85,8 @@ def current_root_task():
         raise RuntimeError("must be called from async context")
 
 
-def reschedule(task, next_send=_NO_SEND):
+def reschedule(task: trio.lowlevel.Task, next_send: Outcome[Any]=_NO_SEND
```

Comment:
[commenting CI check run]
` trio/_core/_generated_run.py:88: error: Cannot determine type of "_NO_SEND"  [has-type]`
This error seems only to pop up when running `mypy -m trio` as opposed to `mypy trio`, so it seems to be a case of circular imports or something. I think we can transition to running the latter in a bit, but for now you can just `type: ignore[misc]` it and add a comment about it.

Reply:
This should just be because `Outcome` is untyped right now.

Reply:
Huh, weird how it gets fixed by either.